README.md

```diff
@@ -319,7 +319,7 @@ To update Testify to the latest version, use `go get -u github.com/stretchr/test
 Supported go versions
 ==================
 
-We support the three major Go versions, which are 1.8, 1.9 and 1.10 at the moment.
+We support the four major Go versions, which are 1.8, 1.9, 1.10, and 1.11 at the moment.
```

Comment:
Please change leave the official support at three versions, but do shift to 1.11.

Reply:
Agreed, let's not expand our commitment going forward, but as long as we still do support 1.8 we can leave it listed.

Reply:
@georgelesica-wf I want to make sure I understand your comment correctly. You said agreed to only support three versions, but you also stated to keep 1.8 listed, and those two things seem at odds. Can you confirm, do you want this to read that 1.8 to 1.11 is officially supported, or 1.9 to 1.11?

Reply:
After re-reading a few times, I think you meant to keep the statement at supporting the three major versions (1.9 to 1.11), but that 1.8 could stay listed in the build tests. I've updated the PR to say that. If that isn't correct please let me know.

Reply:
Yep, that is exactly what I meant. Sorry, I should have phrased that better. I just didn't want to us to change the commitment to three versions even though we may technically support four or more at a given point in time. Thanks!